src/components/tl/ucp/allreduce/allreduce.c

```diff
@@ -13,13 +13,17 @@ ucc_base_coll_alg_info_t
         [UCC_TL_UCP_ALLREDUCE_ALG_KNOMIAL] =
             {.id   = UCC_TL_UCP_ALLREDUCE_ALG_KNOMIAL,
              .name = "knomial",
-             .desc =
-                 "recursive knomial with arbitrary radix (optimized for latency)"},
+             .desc = "recursive knomial with arbitrary radix (optimized for "
+                     "latency)"},
```

Comment:
Probably not needed, but the formatter was run, so some changes in here take the new format into account.
We can remove to not have the extra diff., or leave